pom.xml

```diff
@@ -462,16 +437,16 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-project-info-reports-plugin</artifactId>
-                <version>2.4</version>
+                <version>3.4.1</version>
```

Comment:
This is inherited from parent. We should avoid plugin definition if not necessary.

Reply:
OK, version from the parent is the same so i removed it.